catalog/sat-io/projects_sat-io_open-datasets_us-drought-monitor.jsonnet

```diff
@@ -0,0 +1,102 @@
+local id = 'projects/sat-io/open-datasets/us-drought-monitor';
+local subdir = 'sat-io';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  // TODO: remove 'gee:skip_indexing'
+  'gee:skip_indexing': true,
+  'gee:user_uploaded': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'United States Drought Monitor',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    The U.S. Drought Monitor is a map released every Thursday, showing parts of the U.S. that are in drought. The map uses five classifications: abnormally dry (D0), showing areas that may be going into or are coming out of drought, and four levels of drought: moderate (D1), severe (D2), extreme (D3) and exceptional (D4). The Drought Monitor has been a team effort since its inception in 1999, produced jointly by the National Drought Mitigation Center (NDMC) at the University of Nebraska-Lincoln, the National Oceanic and Atmospheric Administration (NOAA), and the U.S. Department of Agriculture (USDA). The NDMC hosts the web site of the drought monitor and the associated data, and provides the map and data to NOAA, USDA and other agencies. It is freely available at droughtmonitor.unl.edu.
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'drought',
+    'ndmc',
+    'noaa',
+    'usda',
+  ],
+  providers: [
+    ee.producer_provider('National Drought Mitigation Center', 'https://drought.unl.edu/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2000-01-04T00:00:00Z', '2023-09-17T00:00:00Z'),
```

Comment:
If the dataset is periodically updated, you can leave end_date as null and we'll automatically populate it by checking the catalog to see what's the system time on the latest asset. (This is one of the places where we actually do just read the thing!)